### PR TITLE
GitLens 14.0 Release Notes Update

### PR DESCRIPTION
**Pull request by axosoft-ramint**
Adds release notes and image content for 14.0 to the docs.

### CODE REVIEW

**Comment by axosoft-ramint** [comment]
Creating a draft so we can collaborate on it. Will update as material becomes available and move it out of draft once it's ready.

File: gitlens/gitlens-release-notes-current.md [hunk 1]
```diff
@@ -19,6 +19,173 @@ Check out our [Changelog](https://github.com/gitkraken/vscode-gitlens/blob/main/
...
+
+### Changes Column
+The Changes column is now visible by default. Previously, this column was hidden by default. No more navigating through settings to enable the Changes column – it's right there when you need it, providing immediate insights into the modifications made in each commit. This change is part of our commitment to making GitLens as user-friendly and accessible as possible.
+
+The changes column now also includes a file count. The file account allows you to see at a glance the number of files involved in each commit, helping you quickly grasp the scale of each change. This new count includes files that have been added, modified, or deleted in each commit.
+
+### Branch/Tag Column
+In our ongoing efforts to enhance the user experience, we've made significant improvements to the tooltips on the Branch/Tag icons. Now, they're more uniform and descriptive, providing clearer and more consistent information. This update will make navigating through your branches and tags easier and more intuitive than ever before.
```
Comment (axosoft-ramint) [inline gitlens/gitlens-release-notes-current.md:152]:
```suggestion
In our ongoing efforts to enhance the user experience, we've made some minor improvements to the tooltips on the Branch/Tag icons. Now, they're more uniform and descriptive, providing clearer and more consistent information. This update will help users who are new to the graph quickly grasp what each icon in this column represents.
```

File: gitlens/gitlens-release-notes-current.md [hunk 2]
```diff
@@ -19,6 +19,173 @@ Check out our [Changelog](https://github.com/gitkraken/vscode-gitlens/blob/main/
...
+
+We've resolved an issue that was sometimes preventing Branch/Tag nodes from expanding when hovered over. This bug fix ensures a smoother and more reliable interaction with the Branch/Tag nodes, enhancing the overall usability and functionality of GitLens. We're continuously working to improve GitLens and your feedback is vital in helping us fix these bugs. Thank you for your support!
```
Comment (axosoft-ramint) [inline gitlens/gitlens-release-notes-current.md:154]:
```suggestion
```

This bugfix doesn't warrant a mention in this section imo - in the changelog is enough.

File: gitlens/gitlens-release-notes-current.md [hunk 3]
```diff
@@ -19,6 +19,173 @@ Check out our [Changelog](https://github.com/gitkraken/vscode-gitlens/blob/main/
...
+
+### Scroll Markers
+Scroll markers now display better alignment with the scrollbar. This enhancement ensures that the scroll markers are accurately placed, providing a smoother scrolling experience and making it easier for you to navigate.
+
+We've also introduced toggle on and off options for specific scroll markers, including toggles for local branches, remote branches, stashes, and tags. With these additions, you now have the ability to toggle between different scroll marker types. This feature offers you greater control and customization over how you visualize and navigate your repository's commit history. 
+
+You can easily access these new options by clicking on the graph icon dropdown in the top right of the Activity Minimap, and select the scroll marker type that best suits your needs.
```
Comment (axosoft-ramint) [inline gitlens/gitlens-release-notes-current.md:161]:
```suggestion
We've introduced toggle options for specific scroll markers on the settings menu in the upper right of the graph header and the dropdown menu on the right end of the Activity Minimap, including toggles for local branches, remote branches, stashes, and tags. You can now easily toggle individual marker types without having to go into advanced settings.
```

File: gitlens/gitlens-release-notes-current.md [hunk 4]
```diff
@@ -19,6 +19,173 @@ Check out our [Changelog](https://github.com/gitkraken/vscode-gitlens/blob/main/
...
+
+The changes column now also includes a file count. The file account allows you to see at a glance the number of files involved in each commit, helping you quickly grasp the scale of each change. This new count includes files that have been added, modified, or deleted in each commit.
+
+### Branch/Tag Column
+In our ongoing efforts to enhance the user experience, we've made significant improvements to the tooltips on the Branch/Tag icons. Now, they're more uniform and descriptive, providing clearer and more consistent information. This update will make navigating through your branches and tags easier and more intuitive than ever before.
+
+We've resolved an issue that was sometimes preventing Branch/Tag nodes from expanding when hovered over. This bug fix ensures a smoother and more reliable interaction with the Branch/Tag nodes, enhancing the overall usability and functionality of GitLens. We're continuously working to improve GitLens and your feedback is vital in helping us fix these bugs. Thank you for your support!
+
+### Scroll Markers
```
Comment (axosoft-ramint) [inline gitlens/gitlens-release-notes-current.md:156]:
```suggestion
### Scroll and Minimap Markers
```

We added individual marker toggles for both the scrollbar and minimap.

**Review by axosoft-ramint — COMMENTED** [review]
Took a review pass on it. See comments below

**Comment by axosoft-ramint** [comment]
> FYI I added the rest of the images from the help center here so they do not need to be included in this PR (unless there are updated images): [6dba34b](https://github.com/gitkraken/gitlens-docs/commit/6dba34b31c98a10f6b1fb4fb42e89893735f0829)

@jkelroy I'm not seeing your changes from that commit. Did you push it to the remote? I rebased it on top of `main` so you may need to cherry-pick that commit on top.

**Comment by jkelroy** [comment]
@axosoft-ramint I have only pushed them to my remote - you should be able to see them still. If it is easier to add the images here instead, that is not a problem.

**Pull request merged**